app/lib/utils/translations/translations_account.dart

```diff
@@ -216,6 +241,16 @@ class TranslationsFrAccount implements TranslationsAccount {
   String get uosSigner => 'Signataire';
   @override
   String get uosTitle => 'Signature hors ligne';
+  @override
+  String get welcome => 'Bienvenue';
+  @override
+  String get pinError => 'Le code PIN est incorrect';
```

Comment:
This one is tricky, but the more French way would be the following. :)

```suggestion
  String get pinError => 'Le NIP est incorrect';
```